Build failure: qgis build is failing during sip-build

### Steps To Reproduce

Steps to reproduce the behavior:
1. with latest master do `nix-build -A qgis`

### Build log

```
pyproject.toml: line 7: using '[tool.sip.metadata]' to specify the project metadata is deprecated and will be removed in SIP v7.0.0, use '[project]' instead
Querying qmake about your Qt installation...
These bindings will be built: gui.
Generating the gui bindings...
Generating the _gui .api file...
Generating the _gui .pyi file...
sip-build: An internal error occurred...
Traceback (most recent call last):
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/bin/.sip-build-wrapped", line 9, in <module>
    sys.exit(main())
             ^^^^^^
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/tools/build.py", line 37, in main
    handle_exception(e)
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/exceptions.py", line 83, in handle_exception
    raise e
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/tools/build.py", line 34, in main
    project.build()
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/project.py", line 245, in build
    self.builder.build()
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/builder.py", line 46, in build
    self._generate_bindings()
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/builder.py", line 273, in _generate_bindings
    buildable = bindings.generate()
                ^^^^^^^^^^^^^^^^^^^
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/bindings.py", line 220, in generate
    output_code(spec, self, project, buildable)
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/generator/outputs/code.py", line 60, in output_code
    _module_code(spec, bindings, project, py_debug, buildable)
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/generator/outputs/code.py", line 512, in _module_code
    _virtual_handler(sf, spec, handler)
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/generator/outputs/code.py", line 4530, in _virtual_handler
    sipres_value = _call_default_ctor(spec, ctor)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/generator/outputs/code.py", line 4254, in _call_default_ctor
    class_type = fmt_argument_as_cpp_type(spec, arg.definition)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/kaf0lwx9czgl3xcl3bxicl3apx8gzvj0-python3.11-sip-6.8.0/lib/python3.11/site-packages/sipbuild/generator/outputs/formatters/argument.py", line 46, in fmt_argument_as_cpp_type
    original_typedef = arg.original_typedef
                       ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'WrappedClass' object has no attribute 'original_typedef'
ninja: build stopped: subcommand failed
```

### Additional context

Hydra job: https://hydra.nixos.org/build/246593929

### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

Notify @NixOS/geospatial 

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/sip/default.nix
+++ b/pkgs/development/python-modules/sip/default.nix
@@ -9,6 +9,7 @@
 , ply
 , toml
 , tomli
+, poppler-qt5
 }:
 
 buildPythonPackage rec {
@@ -35,6 +36,10 @@ buildPythonPackage rec {
   doCheck = false;
 
   pythonImportsCheck = [ "sipbuild" ];
+
+  passthru.tests = {
+    inherit poppler-qt5;
+  };
 
   meta = with lib; {
     description = "Creates C++ bindings for Python modules";

--- a/pkgs/development/python-modules/sip/default.nix
+++ b/pkgs/development/python-modules/sip/default.nix
@@ -13,13 +13,13 @@
 
 buildPythonPackage rec {
   pname = "sip";
-  version = "6.8.0";
+  version = "6.8.1";
 
   format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-LtGQSCDLZhtyB+sdzPrr7BpUY9ytkDukSK0ZRVAtCJw=";
+    hash = "sha256-MALfQV4WisP/45OULbxxMcuCreUAAOFSb0aoit4m9Zg=";
   };
 
   nativeBuildInputs = [

